src/containers/shared/components/Transaction/types.ts

```diff
@@ -76,3 +76,19 @@ export interface TransactionMapping {
   TableDetail?: TransactionTableDetailComponent
   parser: TransactionParser
 }
+
+export interface TransactionCommonFields {
+  Account: string
+  TransactionType: string
+  Fee: string
+  Sequence: number
+  AccountTxnID?: string
+  Flags?: number
+  LastLedgerSequence?: number
+  Memos?: object[]
+  Signers?: object[]
+  SourceTag?: number
+  SignerPubKey?: string
+  TicketSequence?: number
+  TxnSignature?: string
+}
```

Comment:
@ckniffen for something like this would it make sense to import the types from xrpl.js to avoid duplication?

Reply:
Yeah.  I want to do that.  Need to play around with getting just want we want out of xrpl.js and tree shaking the rest.  Will need some experimentation